src/ol_infrastructure/applications/open_metadata/__main__.py

```diff
@@ -0,0 +1,180 @@
+network_stack = StackReference(f"infrastructure.aws.network.{stack_info.name}")
+policy_stack = StackReference("infrastructure.aws.policies")
+vault_stack = StackReference(f"infrastructure.vault.operations.{stack_info.name}")
+consul_stack = StackReference(f"infrastructure.consul.operations.{stack_info.name}")
+mitodl_zone_id = dns_stack.require_output("odl_zone_id")
+apps_vpc = network_stack.require_output("applications_vpc")
+open_metadata_environment = f"operations-{stack_info.env_suffix}"
+aws_config = AWSBase(
+    tags={"OU": "operations", "Environment": open_metadata_environment},
+)
+consul_provider = get_consul_provider(stack_info)
+
+consul_security_groups = consul_stack.require_output("security_groups")
+aws_account = get_caller_identity()
+
+open_metadata_role = iam.Role(
+    "open-metadata-poc-instance-role",
+    assume_role_policy=json.dumps(
+        {
+            "Version": "2012-10-17",
+            "Statement": {
+                "Effect": "Allow",
+                "Action": "sts:AssumeRole",
+                "Principal": {"Service": "ec2.amazonaws.com"},
+            },
+        }
+    ),
+    name=f"open-metadata-instance-role-{stack_info.env_suffix}",
+    path="/ol-infrastructure/open-metadata-role/",
+    tags=aws_config.tags,
+)
+
+open_metadata_server_security_group = ec2.SecurityGroup(
+    f"open-metadata-server-security-group-{stack_info.env_suffix}",
+    name=f"open-metadata-server-security-group-{stack_info.env_suffix}",
+    ingress=[
+        # ec2.SecurityGroupIngressArgs(
+        # ),
+        ec2.SecurityGroupIngressArgs(
+            protocol="tcp",
+            from_port=8011,
+            to_port=8011,
+            cidr_blocks=["0.0.0.0/0"],
+            description="Allow mike and matt to connect to http.",
+        ),
+    ],
+    egress=default_egress_args,
+    vpc_id=apps_vpc["id"],
+)
+
+open_metadata_database_security_group = ec2.SecurityGroup(
+    f"open-metadata-database-security-group-{stack_info.env_suffix}",
+    name=f"open-metadata-database-security-group-{stack_info.env_suffix}",
+    description="Access control for the open metadata database.",
+    ingress=[
+        ec2.SecurityGroupIngressArgs(
+            security_groups=[
+                open_metadata_server_security_group.id,
+                consul_security_groups["consul_server"],
+                vault_stack.require_output("vault_server")["security_group"],
+            ],
+            protocol="tcp",
+            from_port=DEFAULT_POSTGRES_PORT,
+            to_port=DEFAULT_POSTGRES_PORT,
+            description="Access to postgres from open metadata servers.",
+        ),
+        ec2.SecurityGroupIngressArgs(
+            security_groups=[],
+            protocol="tcp",
+            from_port=DEFAULT_POSTGRES_PORT,
+            to_port=DEFAULT_POSTGRES_PORT,
+            cidr_blocks=["73.218.126.92/32", "98.110.175.231/32"],
+            description="Allow mike and matt to connect to postgres.",
```

Comment:
You must have taken this from the semantic chart? You'll want to change the CIDR block(s) to be the EKS pod address space, which should be an output of the network stack  like `infrastructure.aws.network.CI` or something.